### PR TITLE
added topograph sort for dgraph so it shows order

### PR DESCRIPTION
**Pull request by MizanulH489**
I fixed it up and changed it a bit so it shows the order of all the classes in the topography sort and put it into a list, main issue is runtime which might be just for my device.
image

### CODE REVIEW

File: schedule-works-be/dgraph.py [hunk 1]
```diff
@@ -1,67 +1,32 @@
...
     """
-    Determine a given course's dependency list. See make_priority_queue() to use properly.
-
-    Wrapper function to clarify args used during recursion.
-    Functions best when courses are fed in order from least expected dependency to most.
-
+    Return a list of nodes in topological order.
 
     Args:
-    G: A directed graph containing courses and edges that point towards a courses prequisite.
```
Comment (MMazenA) [inline schedule-works-be/dgraph.py:16]: Why'd you remove all these functions here

File: schedule-works-be/dgraph.py [hunk 2]
```diff
@@ -1,67 +1,32 @@
-"""Functions for interacting with directed graph structure."""
```
Comment (MMazenA) [inline schedule-works-be/dgraph.py:1]: We should keep all of this stuff and just implement the new method

File: schedule-works-be/dgraph.py [hunk 3]
```diff
@@ -98,8 +63,9 @@ def main():
         ]
     )
     show_graph(graph)
-    make_priority_queue(graph)
 
+    topo_order = topological_sort(graph)
```
Comment (MMazenA) [inline schedule-works-be/dgraph.py:67]: This doesn't work for me I got an error

**Review by MMazenA — CHANGES_REQUESTED** [review]

**Comment by MMazenA** [comment]
The picture you sent is of my code output, we need an array or queue portraying a topological sort not rankings, we can create a rankings using both later.

File: schedule-works-be/dgraph.py (hunk 1 above)
Comment (MizanulH489) [reply schedule-works-be/dgraph.py:16]: I had errors when I kept it, and my topo function sort of did it's role for them. I think I should've kept it tbh

File: schedule-works-be/dgraph.py (hunk 2 above)
Comment (MizanulH489) [reply schedule-works-be/dgraph.py:1]: The topological_sort method I added?

**Review by MMazenA — APPROVED** [review]
looks good to me

**Pull request merged**